source/key-concepts/periodsinstants.md

```diff
@@ -13,11 +13,11 @@ _Example: the 15th June 2015._
 _Example: a month ("July 2015"), a year ("2015"), several months ("July and August 2015") or the eternity._
 
 
-The smallest unit for OpenFisca periods is the **month**. Therefore:
+The smallest unit for OpenFisca periods is the **day**. Therefore:
 
 - All periods are presumed to start on the first day of their first month.
-- A period cannot be smaller than a month.
+- A period cannot be smaller than a day.
```

Comment:
Same issue that reported previously on another section

We could use the new phrasing:

```
The smallest unit for OpenFisca periods is the **day**.  
Larger periods, like `month` or `year`, are presumed to start on the first day of their first month.
```

Reply:
Rephrased.